src/com/parse4cn1/command/ParseUploadCommand.java

```diff
@@ -54,6 +54,7 @@ void setUpRequest(ConnectionRequest request) throws ParseException {
 
         if (contentType != null) {
             request.addRequestHeader(ParseConstants.HEADER_CONTENT_TYPE, contentType);
+            addHeader(ParseConstants.HEADER_CONTENT_TYPE, contentType);
```

Comment:
[line](https://github.com/sidiabale/parse4cn1/blob/master/src/com/parse4cn1/command/ParseCommand.java#L102) overwrite the "request" headers using the local variable  "headers" which is set to [default](https://github.com/sidiabale/parse4cn1/blob/master/src/com/parse4cn1/command/ParseCommand.java#L205)

So when you save a ParseFile the content type is always json

```
http://localhost:1337/parse/files/uidhui2.jpg

X-Parse-Client-Key=[master]
User-Agent=[Mozilla/5.0 (iPod; U; CPU iPhone OS 4_3_3 like Mac OS X; en-us) AppleWebKit/533.17.9 (KHTML, like Gecko) Version/5.0.2 Mobile/8J2 Safari/6533.18.5]
X-Parse-Application-Id=[myAppId]
Content-Type=[application/json]
```